docs/guides/src/main/server/configuration.adoc

```diff
@@ -162,6 +162,24 @@ To add the initial admin user using environment variables, set `KEYCLOAK_ADMIN`
 Keycloak uses them at the first startup to create an initial user with administration rights.
 Once the first user with administrative rights exists, you can use the UI or the command line tool `kcadm.[sh|bat]` to create additional users.
 
+== Using placeholders
+
+You are able to use placeholders to resolve the configuration option value from environment variables.
+
+.Using a placeholder to resolve the host from an environment variable
+[source, bash]
+----
+db-url-host="${r"${MY_DB_HOST}"}"
```

Comment:
these quotes are duplicated during merge, no? shouldn't it be `$r{"${MY_DB_HOST}"}`? Looking at the html output it now looks like this:
![Screenshot 2022-04-14 at 12 59 27](https://user-images.githubusercontent.com/89905860/163378584-055c2378-cdd7-411b-89a2-6f50dbfc0f93.png)
which is wrongly double-quoted (or do I miss sth?)